.ci/docker/Dockerfile_dswx_hls

```diff
@@ -14,7 +14,9 @@
 # Author: Scott Collins
 
 # TODO: could this become an argument?
-FROM artifactory-fn.jpl.nasa.gov:16001/gov/nasa/jpl/opera/adt/opera/dswx_hls:interface
+# FROM artifactory-fn.jpl.nasa.gov:16001/gov/nasa/jpl/opera/adt/opera/dswx_hls:interface
+# TODO: Temporary work around while access issues and workarounds are being figured out.
+FROM artifactory-fn.jpl.nasa.gov:16001/gov/nasa/jpl/opera/sds/adt/dswx_hls:interface
```

Comment:
@collins-jpl @JimHofman - what are your thoughts about pushing this binary out to GitHub Packages such that this code is testable in open source?